src/foam/box/Message.js

```diff
@@ -52,6 +52,7 @@ foam.CLASS({
         RemoteException wrapper = new RemoteException();
         wrapper.setId(t.getClass().getName());
         wrapper.setMessage(t.getMessage());
+        wrapper.setServerObject(t);
```

Comment:
Rather than store the exception on a new property on `RemoteException` so you can get it later and get the permission string being checked from it, I think it would be simpler to store the permission string in `Message.localAttributes` and retrieve it from there in `CapabilityInterceptBox`. Then we don't need the `serverObject` property.

Reply:
That makes sense, implementing this now

Reply:
Passing the Throwable instead of permission string, because the latter requires special handling of AuthorizationException inside Message.replyWithException. This still makes CapabilityInterceptBox look a lot cleaner though